dot/sync/disjoint_block_set.go

```diff
@@ -90,6 +103,35 @@ func newDisjointBlockSet(limit int) *disjointBlockSet {
 	}
 }
 
+func (s *disjointBlockSet) start(ctx context.Context) {
+	timer := time.NewTimer(clearBlocksInterval)
+
+	go func() {
+		for {
+			select {
+			case <-timer.C:
+				s.clearBlocks()
```

Comment:
if this function should run every minute, then you should change from `time.NewTimer()` to `time.NewTicker()`

Reply:
done!